students/chihun/users/views.py

```diff
@@ -1,3 +1,44 @@
-from django.shortcuts import render
+import json
+from json.decoder           import JSONDecodeError
+from django.http            import JsonResponse
+from django.views           import View
+from django.core.exceptions import ValidationError
 
-# Create your views here.
+
+from users.validation       import validate_email, validate_password
+from users.models           import User
+
+class SignUp(View):
+    def post(self, request):
+        try :
+            data         = json.loads(request.body)
+            name         = data['name']
+            email        = data['email']
+            password     = data['password']
+            phone_number = data['phone_number']
+            blog_url     = data.get('blog_url')  # null = True
+
+            validate_email(email)
+            validate_password(password)
+            
+            if User.objects.filter(email = email).exists():
+                return JsonResponse({'MESSAGE':'ALREADY_EXISTS_EMAIL'}, status=400)
```

Comment:
매칭되는 자원(record)이 없는 경우에는 `status code 404`를 반환합니다.

```suggestion
                return JsonResponse({'MESSAGE':'ALREADY_EXISTS_EMAIL'}, status=404)
```